### PR TITLE
feat(web): default product rework

### PR DESCRIPTION
**Pull request by m8vago**
The first created version in a product gets default badge
Setting default version now only possible from the version list page
Add new animal friends to our icon repertoire (thanks to uxwing.com)

### CODE REVIEW

File: web/crux-ui/src/styles/global.css [hunk 1]
```diff
@@ -12,4 +12,4 @@
 
 html, body {
     font-family: 'Poppins', sans-serif;
-}
+}
```
Comment (robot9706) [inline web/crux-ui/src/styles/global.css:15]: New line 🤔

**Review by robot9706 — APPROVED** [review]
NL, otherwise LGTM 😎

**Comment by polaroi8d** [comment]
Actual: 
image

Expected:
image

File: protobuf/proto/crux.proto [hunk 2]
```diff
@@ -55,6 +55,7 @@ service CruxProductVersion {
   rpc CreateVersion(CreateVersionRequest) returns (CreateEntityResponse);
```
Comment (polaroi8d) [inline protobuf/proto/crux.proto:55]: Different formatting on this file again..?

File: web/crux-ui/playwright.config.ts [hunk 3]
```diff
@@ -5,63 +5,63 @@ const baseURL = process.env.E2E_BASE_URL || 'http://172.17.0.1:8000'
```
Comment (polaroi8d) [inline web/crux-ui/playwright.config.ts:5]: Formatting again? We have to talk about this in person.

**Comment by polaroi8d** [comment]
Found this too:

image

File: web/crux-ui/src/elements/dyo-badge.tsx [hunk 4]
```diff
@@ -3,14 +3,17 @@ import Image from 'next/image'
...
 
 const DyoBadge = (props: DyoBadgeProps) => {
-  const { icon } = props
+  const { icon, large } = props
+
+  const size = large ? 32 : 24
 
   const { t } = useTranslation('badges')
 
-  return <Image src={`/badges/${icon}.svg`} alt={t(`${icon}`)} width={24} height={24} />
```
Comment (polaroi8d) [inline web/crux-ui/src/elements/dyo-badge.tsx:13]: Please use the default 24px size. It was okey.

File: protobuf/proto/crux.proto (hunk 2 above)
Comment (m8vago) [reply protobuf/proto/crux.proto:55]: what you mean?

**Comment by m8vago** [comment]
@polaroi8d  check again ples

**Comment by polaroi8d** [comment]
> @polaroi8d check again ples

The pictures in nodes still not 24 pixels.

File: web/crux-ui/src/elements/dyo-badge.tsx (hunk 4 above)
Comment (polaroi8d) [reply web/crux-ui/src/elements/dyo-badge.tsx:13]: Still problem.

**Review by polaroi8d — APPROVED** [review]

**Pull request merged**